sqlmesh/cli/main.py

```diff
@@ -376,5 +376,44 @@ def migrate(ctx: click.Context) -> None:
     ctx.obj.migrate()
 
 
+@cli.command("prompt")
+@click.argument("prompt")
+@click.option(
+    "-e",
+    "--evaluate",
+    is_flag=True,
+    help="Evaluate the generated SQL query and display the results.",
+)
+@click.option(
+    "-t",
+    "--temperature",
+    type=float,
+    help="Sampling temperature. Default: 0.7",
```

Comment:
Want to provide a bit more of an explanation? Maybe just use the framing that Bing uses: https://arstechnica.com/information-technology/2023/03/microsoft-equips-bing-chat-with-multiple-personalities-creative-balanced-precise/

Reply:
So 0 - price, .5 - balanced, 1 - creative.